src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/models/State.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.indexstatemanagement.models
+
+import org.elasticsearch.common.lucene.uid.Versions
+import org.elasticsearch.common.xcontent.ToXContent
+import org.elasticsearch.common.xcontent.XContentBuilder
+import org.elasticsearch.common.xcontent.XContentParser
+import org.elasticsearch.common.xcontent.XContentParser.Token
+import org.elasticsearch.common.xcontent.XContentParserUtils.ensureExpectedToken
+import java.io.IOException
+
+data class State(
+    val id: String = NO_ID,
+    val version: Long = NO_VERSION,
+    val stateName: String,
+    val actions: List<Map<String, Any>>, // TODO: Implement List<Action>
+    val transitions: List<Map<String, Any>> // TODO: Implement List<Transition>
+) : ToXContent {
+
+    fun toXContent(builder: XContentBuilder): XContentBuilder =
```

Comment:
Shouldn't need this on State; if you saw this from Policy it was because of the PutPolicy API

Reply:
Removed.